src/engine/GroupBy.cpp

```diff
@@ -849,7 +839,7 @@ GroupBy::isSupportedAggregate(sparqlExpression::SparqlExpression* expr) {
   if (hasType<MaxExpression>(expr)) return H{MAX};
   if (hasType<SumExpression>(expr)) return H{SUM};
   if (auto val = hasType<GroupConcatExpression>(expr)) {
```

Comment:
Not sure if it would be more readable to just inline the dynamic cast here

Reply:
As I see it absolutely (especially since there is only one usage, and the name is now off anyway.